### ISSUE TITLE
ui.input validation will remain space after it's value was corrected

### ISSUE THREAD

**Issue by atticus-lv** [zauberzeug/nicegui#3013]
### Description

https://github.com/zauberzeug/nicegui/assets/64118610/e577f350-0f23-4cad-8fdf-6425d62c5e78

Code:
```python
from nicegui import ui

with ui.row():
    ui.input('Version', value='4.1.1', placeholder='4.1.1',
             validation={
                 'error str': lambda v: v.count('.') == 2 and v.replace(
                     '.',
                     '').isdigit(),
             },) \
        .style('width: 200px') \

    ui.input('Version2', value="1.3.0", placeholder='1.3.0',
             validation={
                 'error str': lambda v: v.count('.') == 2 and v.replace(
                     '.',
                     '').isdigit(),
             },)\
        .style('width: 200px') \

    ui.select(value='stable',options =
        {'stable': 'Stable',
         'beta': 'Beta',
         'alpha': 'Alpha'},)

ui.input('Name', value = '').style('width: 500px')

ui.run(native=True)
```

**Comment by rodja** [comment]
Use the Quasar prop `hide-bottom-space` as described in the Quasar documentaion.

**Comment by atticus-lv** [comment]
thanks it works...maybe this should set by default

**Comment by falkoschindler** [comment]
This has been discussed in #2484 and fixed in PR #2492. But because it breaks existing layouts, it is scheduled to be released in version 2.0.
